app/storybook/documentation/ui-kit/api/DataPoolInput.mdx

```diff
@@ -0,0 +1,10 @@
+import { ArgTypes, Meta } from '@storybook/blocks'
+import * as DataPoolInputStories from '../../../../../packages/ui-kit/src/helpers/graphql/stories/DataPoolInput.stories'
+
+<Meta title="API/DataPoolInput" of={DataPoolInputStories} />
+
+# `DataPoolInput`
+
+**Note:** The properties below are mutually exclusive due to the `@oneOf` directive. Only one of these properties should be provided for a valid `DataPoolInput`.
```

Comment:
UI Kit users don't necessarily know or care about the `@oneOf` GraphQL directive. They may not even know we use GraphQL under the hood.
```suggestion
**Note:** The properties below are mutually exclusive. Only one of these properties should be provided for a valid `DataPoolInput`.
```